### DISCUSSION TITLE
Is there an intro to the svelte codebase?

### DISCUSSION THREAD

**Discussion by csenio** [sveltejs/kit#4483] [Q&A]
I'd like to start contributing but I'm having a rough time understanding what's going on in the codebase. I've also never worked on a project such as svelte kit so the terminology is unfamiliar.
Do we have any documentation or discussions somewhere that can help me get into the codebase? 
Mostly curious about what folder does what and where the entry point is. I've been trying to see what happens under the hood when I run `yarn dev` on my website, but couldn't find the corresponding code so far.

**Comment by benmccann** [comment]
https://github.com/sveltejs/kit/blob/master/CONTRIBUTING.md

don't use `yarn`. you need `pnpm`
